coq/coq-indent.el

```diff
@@ -365,7 +365,7 @@ and return nil."
   "Test if between point and previous command is empty.
 Comments are ignored, of course."
   (let ((end (point))
-        (start (coq-find-not-in-comment-backward "[^[:space:]]")))
+        (start (coq-find-not-in-comment-backward "(Time\\|Fail)?[^[:space:]]")))
```

Comment:
first remark shouldn't those parenthesis be escaped \\(Time\\|Fail\\)?
second remark: I don't see how this could fix things, if we find Time we must recrsively call coq-empty-p to check if there is nothing significant before Time (in which case Time was not the Time command but maybe some coq constant named Time).
The function first looks for something that is significant (non space, non comment) and decides if this is the end of a previous command (which is done recursively). If yes then the command was empty.
`(coq-find-not-in-comment-backward "(Time\\|Fail)?[^[:space:]]")` tries too hard to 

I

Reply:
Sorry, I got this all wrong. I'll try again tomorrow.